### PR TITLE
Add presubmit config.

### PR DESCRIPTION
**Pull request by xingao267**
Fixes https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/issues/10

### CODE REVIEW

**Comment by MartinPetkov** [comment]
The files LGTM but where does it run? Do we also need a `.github/workflows` file, or a cloudbuild.yaml somewhere?

**Comment by xingao267** [comment]
> The files LGTM but where does it run? Do we also need a `.github/workflows` file, or a cloudbuild.yaml somewhere?

Check the Trigger in the Checks. It's running on GCB and it's using the build/presubmit.yaml.

**Comment by MartinPetkov** [comment]
> > The files LGTM but where does it run? Do we also need a `.github/workflows` file, or a cloudbuild.yaml somewhere?
> 
> Check the Trigger in the Checks. It's running on GCB and it's using the build/presubmit.yaml.

I see, thank you.

File: build/go_check.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+# Check format
+f=$(gofmt -l .)
+if [[ "${f}" ]]; then
+  echo "The following files are not formatted:"
+  echo "${f}"
+  exit 1
+fi
+
+# Check dependencies up-to-date
```
Comment (MartinPetkov) [inline build/go_check.sh:26]: Maybe remove this block, given that we have Dependabot enabled.

File: build/presubmit.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+- name: "gcr.io/cloud-builders/go"
+  entrypoint: "sh"
```
Comment (MartinPetkov) [inline build/presubmit.yaml:17]: Why not just make the entrypoint be the check script?

Comment (xingao267) [reply build/presubmit.yaml:17]: That doesn't seem to work. It gives me `standard_init_linux.go:211: exec user process caused "exec format error"`

File: build/go_check.sh (hunk 1 above)
Comment (xingao267) [reply build/go_check.sh]: Done.

**Pull request merged**